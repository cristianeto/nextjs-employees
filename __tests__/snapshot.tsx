/* eslint-disable import/no-extraneous-dependencies */
import { render } from '@testing-library/react';
import { RouterContext } from 'next/dist/shared/lib/router-context';
import { createMockRouter } from '@mocks';
import Home from '@pages/index';

it('renders homepage unchanged', () => {
  const { container } = render(
    <RouterContext.Provider value={createMockRouter({})}>
      <Home />
    </RouterContext.Provider>,
  );
  expect(container).toMatchSnapshot();
});
