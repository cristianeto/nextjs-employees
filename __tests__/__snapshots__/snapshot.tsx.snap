// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <nav>
      <ul>
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/auth/login"
          >
            Login
          </a>
        </li>
      </ul>
    </nav>
    <h1>
      Home
    </h1>
  </div>
</div>
`;
